sdk/vaa/structs_test.go

```diff
@@ -335,6 +337,7 @@ func TestChainId_String(t *testing.T) {
 		{input: 45, output: "worldchain"},
 		{input: 46, output: "ink"},
 		{input: 47, output: "hyperevm"},
+		{input: 49, output: "movement"},
```

Comment:
ditto, monad is missing here

Reply:
Don't want to cloud this PR with that stuff. Will do a follow-on PR.